src/index.html

```diff
@@ -2,13 +2,50 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="logotip"
+        class="logo__image">
+      </a>
+      <nav class="navbar">
+        <ul class="nav__list">
+          <li class="nav__text">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__text">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__text">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__text"> <a href="#" class="nav__link"
+            data-qa="hover">Laptops & computers</a>
+          </li>
```

Comment:
let's align your code 
```suggestion
          <li class="nav__text"> 
             <a
                href="#"
                class="nav__link" 
                data-qa="hover"
              >Laptops & computers</a>
          </li>
```